### ISSUE TITLE
Format error with media query

### ISSUE THREAD

**Issue by hstarorg** [ThisIsManta/vscode-stylus-supremacy#10]
```css
@media only screen and (min-width: 0px) and (max-width: 679px){}
```
If stylus code have the above codes, it will be format error: `Found a non-string argument of`

**Comment by ThisIsManta** [comment]
Investigating... Please bear with me.

**Comment by ThisIsManta** [comment]
This has been fixed at version 0.3.4. Please update the extension.

I've made sure that **Stylus Supremacy** generates `@media` queries the same way as the original **Stylus** parser.

Thank you for reporting!

**Comment by hstarorg** [comment]
Very thanks. I have a new issue in #11
